pkg/config/draftconfig.go

```diff
@@ -9,9 +9,10 @@ import (
 
 // TODO: remove Name Overrides since we don't need them anymore
 type DraftConfig struct {
-	DisplayName   string             `yaml:"displayName"`
-	NameOverrides []FileNameOverride `yaml:"nameOverrides"`
-	Variables     []*BuilderVar      `yaml:"variables"`
+	DisplayName         string             `yaml:"displayName"`
+	NameOverrides       []FileNameOverride `yaml:"nameOverrides"`
+	Variables           []*BuilderVar      `yaml:"variables"`
+	FileNameOverrideMap map[string]string  `yaml:"filenameOverrideMap"`
```

Comment:
this looks like we have 3 override related fieldson the draft config. we will need to consolidate them into one implementation in a future PR.